build.gradle.kts

```diff
@@ -264,6 +264,13 @@ tasks {
 
     jacocoTestReport {
         dependsOn(test)
+        dependsOn("itest")
+
+        reports {
+            xml.required.set(false)
```

Comment:
Hmm I'm pretty sure XML is required for the test reporting thingy we have running in GitHub. I think by default is generates XML reports. For that reason it's the HTML reports we don't need I think

Reply:
hmm, ok, this is not really part of this PR, I'll remove this change...
The idea with it is to be able to publish some readable pages and publish these, so that we can check issues easier.

Reply:
Ok!